src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository  extends JpaRepository<Book, Long> {
+    Book findBookByAuthor(String author, Long id);
```

Comment:
Long id attribute in this query is not required. 

```suggestion
    Book findBookByAuthor(String author);
```

Reply:
I removed this param